tests/small/LandIce/FO_GIS/input_fo_humboldt_frosch_fluxdiv.yaml

```diff
@@ -45,26 +46,21 @@ ANONYMOUS:
     LandIce Viscosity:
       Extract Strain Rate Sq: true
       Type: Glen's Law
-      Glen's Law Homotopy Parameter: 0.3e+00
+      Glen's Law Homotopy Parameter: 0.3
       Continuous Homotopy With Constant Initial Viscosity: true
-      Coefficient For Continuous Homotopy: 8.00000000000000000e+00
+      Coefficient For Continuous Homotopy: 8.0
```

Comment:
Thanks for trimming all of these (overly) long decimals! =)

Reply:
Sure. I did that only for the tests that I had to modify. If each of us trim those when working on a particular test at some point we'll get rid of those long decimals. We could make a script to fix them.. but it's not always trivial, and we do not necessarily want all the numbers in the same format.

Reply:
I don't think it's worth the time to write a script. As you said, we can just trim decimals as we work on tests.